src/torManager/torManager.test.ts

```diff
@@ -0,0 +1,116 @@
+/* eslint import/first: 0 */
+import { Tor } from './torManager'
+import { ZBAY_DIR_PATH } from '../constants'
+import { getPorts } from '../utils'
+
+jest.setTimeout(30_000)
+
+test('start and close tor', async () => {
+  const ports = await getPorts()
+  const torPath = `${process.cwd()}/tor/tor`
+  const libPath = `${process.cwd()}/tor`
+  const tor = new Tor({
+    appDataPath: ZBAY_DIR_PATH,
+    torPath: torPath,
+    controlPort: ports.controlPort,
+    socksPort: ports.socksPort,
+    options: {
+      env: {
+        LD_LIBRARY_PATH: libPath,
+        HOME: ZBAY_DIR_PATH
+      },
+      detached: true
+    }
+  })
+  await tor.init()
+  await tor.kill()
```

Comment:
Shouldn't we spy on tor.process.kill (and maybe check if it succeeded) or at least assert that tor.kill didn't throw any error?